cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -111,7 +140,23 @@ export class PassportAuth implements EndpointSecurity {
    * @param passportApi - passport.js instance
    */
   protected setup(passportApi: passport.Passport) {
+    const options = {
+      jwtFromRequest: ExtractJwt.fromAuthHeader(),
+      secretOrKey: 'secret'
```

Comment:
Remember to extract and document that as keeping this `secret` is crucial for general application security.